bitnami/grafana-loki/values.yaml

```diff
@@ -4462,521 +4462,63 @@ tableManager:
-    ##               - key: role
-    ##                 operator: In
-    ##                 values:
-    ##                   - frontend
-    extraIngress: []
-    ## @param promtail.networkPolicy.extraEgress [array] Add extra ingress rules to the NetworkPolicy
-    ## e.g:
-    ## extraEgress:
-    ##   - ports:
-    ##       - port: 1234
-    ##     to:
-    ##       - podSelector:
-    ##           - matchLabels:
-    ##               - role: frontend
-    ##       - podSelector:
-    ##           - matchExpressions:
-    ##               - key: role
-    ##                 operator: In
-    ##                 values:
-    ##                   - frontend
-    ##
-    extraEgress: []
-    ## @param promtail.networkPolicy.ingressPodMatchLabels [object] Labels to match to allow traffic from other pods. Ignored if `promtail.networkPolicy.allowExternal` is true.
+  replicaCount: 1
+  ## @section Alloy container specific parameters
+  ##
+  alloy:
+    ## @param grafanaalloy.alloy.stabilityLevel Minimum stability level of components and behavior to enable. Must be
+    ##  one of "experimental", "public-preview", or "generally-available".
+    stabilityLevel: "generally-available"
+    ## @param grafanaalloy.alloy.listenAddr Address to listen for traffic on. 0.0.0.0 exposes the UI to other containers.
+    listenAddr: 0.0.0.0
+    ## @param grafanaalloy.alloy.uiPathPrefix Base path where the UI is exposed.
+    uiPathPrefix: /
+    ## @param grafanaalloy.alloy.storagePath Path to where Grafana Alloy stores data (for example, the Write-Ahead Log).
+    storagePath: /tmp/alloy
+    ## @param grafanaalloy.alloy.enableReporting Enables sending Grafana Labs anonymous usage stats to help improve Grafana grafanaalloy.alloy.
+    enableReporting: true
+    ## @param grafanaalloy.alloy.mounts.varlog Mount /var/log from the host into the container for log collection.
+    ## @param grafanaalloy.alloy.mounts.dockercontainers Mount /var/lib/docker/containers from the host into the container for log
+    ##
+    mounts:
+      varlog: true
+      dockercontainers: true
+    ## @param grafanaalloy.alloy.clustering.enabled Deploy Grafana Alloy in a cluster to allow for load distribution.
+    ## @param grafanaalloy.alloy.clustering.name Name for the Grafana Alloy cluster. Used for differentiating between clusters.
+    ## @param grafanaalloy.alloy.clustering.portName Name for the port used for clustering, useful if running inside an Istio Mesh
+    ##
+    clustering:
+      enabled: false
+      name: ""
+      portName: http
+    ## @param grafanaalloy.alloy.createSecret Create a Secret with the default configuration for Grafana Alloy to send logs to Loki.
+    ##
+    createSecret: true
+    ## @param grafanaalloy.alloy.existingSecret The name of an existing Secret with your custom configuration for Grafana Alloy. If the grafanaalloy.alloy.createSecret is set, it will create a Secret with this name.
+    ##
+    existingSecret: "grafana-loki-grafanaalloy"
```

Comment:
This is weird, you're both setting `grafanaalloy.alloy.createSecret` and `grafanaalloy.alloy.existingSecret` and you're hardcoding the name.

Reply:
The idea is to set both: notifying Alloy not to create the secret, and asking Loki to create it. Do you suggest any alternative?